cookiecutter.json

```diff
@@ -1,31 +1,36 @@
 {
-  "full_name": "<Your full name>",
-  "email": "<Your email>",
-  "contribution_name": "<Contribution name>",
-  "autora_contribution_type": ["theorist", "experimentalist", "experiment_runner", "synthetic_data", "custom"],
-  "custom_autora_contribution_type": "{% if cookiecutter.autora_contribution_type == 'custom' -%}<custom_autora_contribution_type>{% else -%}N/A - Press Enter to Skip{% endif -%}",
+  "full_name": "Enter your full name",
+  "email": "Enter your e-mail adress",
+  "contribution_name": "Choose a name of your contribution here. This will also be your package name. Choose something short but descriptive",
+  "__contrib_name": "{% if cookiecutter.contribution_name == 'Choose a name of your contribution here. This will also be your package name. Choose something short but descriptive' -%}{% else -%}{{ cookiecutter.contribution_name }}{% endif -%}",
```

Comment:
```suggestion
  "__contrib_name": "{% if cookiecutter.contribution_name == 'Choose a name for your contribution here. This will also be your package name. It's best to choose something short but descriptive' -%}{% else -%}{{ cookiecutter.contribution_name }}{% endif -%}",
```